src/Context/Context.tsx

```diff
@@ -0,0 +1,250 @@
+import React, { createContext, useMemo, useState } from 'react';
+import { Person } from '../types/Person/Person';
+import { Filter } from '../enums/Filter';
+import { useSearchParams } from 'react-router-dom';
+import { SortValues } from '../enums/SortValues';
+
+// #region context
+
+interface PeopleContextType {
+  searchParams: URLSearchParams;
+  people: Person[];
+  filteredPeople: Person[];
+  activeCenturies: string[];
+  loader: boolean;
+  filter: Filter;
+  loadingError: boolean;
+  sexParam: string | null;
+  sortParam: SortValues | null;
+  orderParam: 'desc' | null;
+  query: string;
+  setLoader: (value: boolean) => void;
+  setLoadingError: (value: boolean) => void;
+  setFilter: (filter: Filter) => void;
+  setPeople: (people: Person[]) => void;
+  queryHandler: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  centuriesHandler: (value: string) => void;
+  resetHandler: () => void;
+  sortHandler: (value: SortValues) => void;
+}
+
+export const PeopleContext = createContext<PeopleContextType>({
+  searchParams: new URLSearchParams(),
+  people: [],
+  filteredPeople: [],
+  activeCenturies: [],
+  loader: false,
+  filter: Filter.all,
+  loadingError: false,
+  sexParam: '',
+  sortParam: SortValues.name,
```

Comment:
The initial value for `sortParam` should be `null` instead of `SortValues.name` to match the type definition in `PeopleContextType`.